dream_layer_backend/setup.py

```
@@ -0,0 +1,44 @@
+#!/usr/bin/env python3
+"""
+Setup script for DreamLayer CLI
+"""
+
+from setuptools import setup, find_packages
```

Ruff (0.11.9)

`setuptools.find_packages` imported but unused

Remove unused import: `setuptools.find_packages`

(F401)